locales/client/en.json

```diff
@@ -939,7 +939,9 @@
   "Settings": {
     "title": "LikeCoin ID Settings",
     "label": {
-      "contactUs": "Contact us for {deleteAccount} or {requestReport}"
+      "contactUs": "Contact us for {deleteAccount} or {requestReport}",
+      "deleteAccountPrePopulatedMessage": "I want to delete my account",
+      "requestReportPrePopulatedMessage": "I want to report "
```

Comment:
`I want to request a account data report` ...

Reply:
Updated @williamchong007